backend/accounts/feetarget.go

```diff
@@ -58,10 +59,51 @@ const (
 	// FeeTargetCodeHigh is the high priority fee target.
 	FeeTargetCodeHigh FeeTargetCode = "high"
 
+	// FeeTargetCodeMempoolFastest is the mempool highest priority fee target.
+	FeeTargetCodeMempoolFastest FeeTargetCode = "mFastest"
+
+	// FeeTargetCodeMempoolHalfHour is the mempool half hour fee target.
+	FeeTargetCodeMempoolHalfHour FeeTargetCode = "mHalfHour"
+
+	// FeeTargetCodeMempoolHour is the mempool hour fee target.
+	FeeTargetCodeMempoolHour FeeTargetCode = "mHour"
+
+	// FeeTargetCodeMempoolEconomy is the mempool economy fee target.
+	FeeTargetCodeMempoolEconomy FeeTargetCode = "mEconomy"
+
 	// FeeTargetCodeCustom means that the actual feerate is supplied separately instead of being
 	// estimated automatically.
 	FeeTargetCodeCustom FeeTargetCode = "custom"
 
+	// DefaultMempoolFeeTarget is the default fee target for mempool fees.
+	DefaultMempoolFeeTarget = FeeTargetCodeMempoolHalfHour
+
 	// DefaultFeeTarget is the default fee target.
 	DefaultFeeTarget = FeeTargetCodeNormal
 )
+
+// MempoolSpaceFees contains mempool.space recommended fees API response
+// (https://mempool.space/docs/api/rest#get-recommended-fees)
+type MempoolSpaceFees struct {
+	FastestFee  int64 `json:"fastestFee"`
+	HalfHourFee int64 `json:"halfHourFee"`
+	HourFee     int64 `json:"hourFee"`
+	EconomyFee  int64 `json:"economyFee"`
+	MinimumFee  int64 `json:"minimumFee"`
+}
+
+// GetFeeAmount returns the btcutil.Amount of the fee for the given FeeTargetCode.
+func (fees MempoolSpaceFees) GetFeeAmount(code FeeTargetCode) btcutil.Amount {
```

Comment:
Better name could be `GetFeeRate()`, as this does not return the fee, but the fee rate (sats/vB).

Reply:
Ack, I wasn't really satisfied about the name either